docs/changelogs/CHANGELOG-2.26.md

```diff
@@ -0,0 +1,320 @@
+- Update Velero Helm chart to v1.13.1 ([#13272](https://github.com/kubermatic/kubermatic/pull/13272))
+- Update Velero to v1.14.0 ([#13473](https://github.com/kubermatic/kubermatic/pull/13473))
+
+### Cleanup
+
+- Add SecurityContext to KKP operator/controller-manager containers, including OSM and machine-controller ([#13282](https://github.com/kubermatic/kubermatic/pull/13282))
+- Addon conditions now contain the KKP version that has last successfully reconciled the addon (similar to the Cluster conditions) ([#13519](https://github.com/kubermatic/kubermatic/pull/13519))
+- Addons reconciliation is triggered more consistently for changes to Cluster objects, reducing the overall number of unnecessary addon reconciliations ([#13252](https://github.com/kubermatic/kubermatic/pull/13252))
+- Fix an issue which prohibited users to specify custom values for Cilium system application ([#13276](https://github.com/kubermatic/kubermatic/pull/13276))       
+- Fix misleading errors about undeploying the cluster-backup components from newly created user clusters ([#13403](https://github.com/kubermatic/kubermatic/pull/13403))
+- Replace custom Velero Helm chart with a wrapper around the official upstream chart ([#13488](https://github.com/kubermatic/kubermatic/pull/13488))
+- Replace kubernetes.io/ingress.class annotation with ingressClassName spec field ([#13549](https://github.com/kubermatic/kubermatic/pull/13549))
+- S3-Exporter does not run with root permissions and does not leak credentials via CLI flags anymore ([#13226](https://github.com/kubermatic/kubermatic/pull/13226))
+- Remove deprecated Cilium and Hubble KKP Addons, as Cilium CNI is managed by Applications ([#13229](https://github.com/kubermatic/kubermatic/pull/13229))
+
+
+### Deprecation
+
+- Add `spec.componentsOverride.coreDNS` to Cluster objects, deprecate `spec.clusterNetwork.coreDNSReplicas` in favor of the new `spec.componentsOverride.coreDNS.replicas` field ([#13409](https://github.com/kubermatic/kubermatic/pull/13409))
+
+
+### Miscellaneous
+
+- Compatibility of addons is now automatically tested against previous KKP releases to prevent addons failing to change immutable fields ([#13256](https://github.com/kubermatic/kubermatic/pull/13256))
+- Fix(metrics-server): correct networkpolicy port for metrics-server ([#13438](https://github.com/kubermatic/kubermatic/pull/13438))
+- Metering CronJobs now use a `metering-` prefix; older jobs are automatically removed ([#13200](https://github.com/kubermatic/kubermatic/pull/13200))
+- Reduce number of Helm upgrades in application-installation-controller by tracking changes to Helm chart version, values and templated manifests ([#13121](https://github.com/kubermatic/kubermatic/pull/13121))
+- Add dynamic base id to envoy agent on the user cluster ([#13261](https://github.com/kubermatic/kubermatic/pull/13261))      
+- Update to controller-runtime 0.18.1 ([#13346](https://github.com/kubermatic/kubermatic/pull/13346))
```

Comment:
This can be removed alltogether, the very first item on this page mentions ctrl-runtime 0.19.